applications/NXxas.nxdl.xml

```diff
@@ -74,15 +74,15 @@
               </field>
             </group>
             <group type="NXdetector" name="incoming_beam">
-              <field name="data" type="NX_INT">
+              <field name="data" type="NX_NUMBER">
                 <dimensions rank="1">
                   <dim index="1" value="nP" />
                 </dimensions>
               </field>
             </group>
             <group type="NXdetector" name="absorbed_beam">
-              <field name="data" type="NX_INT">
-                  <doc>mark this field with attribute  ``signal=1``</doc>
+              <field name="data" type="NX_NUMBER">
+                  <doc>This data corresponds to the sample signal.</doc>
```

Comment:
What is the "sample signal"? Does this refer to the `signal` of the `NXsample` default `NXdata`? Or does it simply refer to the signal measured from the physical sample in the beam. Seem redundant: the data field of any NXdetector will most likely have the sample signal. If you want some documentation of this field, I guess you can specify what signal it is (beam intensity measured after the sample, in transmission).

Reply:
The signal from the sample can take many forms:
* transmission
* drain current
* fluorescence
* electron analyser (with various energy discrimination schemes)

The comment is therefore deliberately ambiguous since I didn't want to do a major overhaul of the application definition at this time.

Reply:
ok